tests/integration/backward_compatible/client_test.py

```diff
@@ -33,7 +33,6 @@ def lifecycle_event_collector():
             events = []
 
             def event_collector(e):
-                print(e)
```

Comment:
I guess this was for debugging and forgotten

Reply:
Yes